README.md

```diff
@@ -1,2 +1,407 @@
+│ --split-data        --no-split-data             flag on whether to split data as was done for train [default: no-split-data]                                               │
+│ --help                                          Show this message and exit.                                                                                                │
+╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+```
+
+## ⬇️  Download
+
+This commands enables you to download mesh data from EPMC
+
+### epmc-mesh
+
+```
+
+ Usage: grants-tagger download epmc-mesh [OPTIONS] DOWNLOAD_PATH
+
+╭─ Arguments ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ *    download_path      TEXT  path to directory where to download EPMC data [default: None] [required]                                                                     │
+╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+╭─ Options ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ --year        INTEGER  year to download epmc publications [default: 2020]                                                                                                  │
+│ --help                 Show this message and exit.                                                                                                                         │
+╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+```
+
+# 🧑🏻‍💻  Develop
+
+Install development dependencies via:
+`poetry install --with dev`
+
+## 📖 Data
+
+If you work for Wellcome and have access to our AWS account,
+you easily download the raw data by typing `make sync_data`.
+This will give you access to both the custom science tags
+dataset and the MeSH data. Note that the mesh data is 50+GB.
+If you want to download only the science or mesh data, you
+can do so with `make sync_science_data` and `make sync_mesh_data`
+respectively.
+
+The MeSH data can be downloaded from various places like EPMC.
+Grants tagger currently uses a sample provided from the [BioASQ](http://www.bioasq.org)
+competition that contains tags for approx 14M publications from PubMed.
+
+## 📋 Env variables
+
+You need to set the following variables for sagemaker or sync to work. If you want to
+participate to BIOASQ competition you need to also set some variables.
+
+Variable              | Required for       | Description
```

Comment:
let's remove the variables that sync and sagemaker, we use `dvc` for sync and we dont use sagemaker anymore. lets also remove the sagemaker dependency in case its there and any sagemaker code

Reply:
Removed